### DISCUSSION TITLE
ArtNet Casting

### DISCUSSION THREAD

**Discussion by stonegray** [spensbot/captivate#44] [Ideas]
It looks like only unicast artnet is supported; tested by using 172.255 as the output IP. 

This presents an issue with setups that use multiple nodes in the same universe; as well as link-local, of which I use both. 

I'm currently working around it by using a passthrough universe in QLC+ (127.0.0.1 -> 169.254.255.255)

**Comment by producation** [comment]
I changed `export const PRIMARY_ARTNET_IP = '2.255.255.255'` in the /src/main/engine/connections/art-net/constants.ts file to the IP my Art-Net node is using. Afterwards I rebuilt the project so now I can use my Art-Net node without a passthrough
